IOS@Taipei/IOS@Taipei/Repository/GroupEventRepository.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  GroupEventRepository.swift
+//  IOS@Taipei
+//
+//  Created by kuotinyen on 2019/10/23.
+//  Copyright © 2019 iOS@Taipei. All rights reserved.
+//
+
+import Foundation
+
+enum APIError: Error {
+    case invalidURL
+    case unknown(Error?)
+    case noResponseData
+    case noDataModel
+    case noFBAccessToken
+}
+
+ final class GroupEventRepository {
+    static let shared = GroupEventRepository()
+    private init() {}
+
```

Comment:
Is there any indent misplace?

Reply:
what do you mean?

Reply:
select this area and press control + i and see what happened.